### PR TITLE
Programmes 6571 aria label related episode

### PR DESCRIPTION
**Pull request by cisco87**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6571

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: templates/find_by_pid/episode.html.twig [hunk 1]
```diff
@@ -226,7 +226,7 @@
...
                                             'title_options': {'title_tag': 'h3', 'title_format': 'item::ancestry', 'link_location_track': 'episode_suggested_item_' ~ loop.index },
                                             'body_options': {
-                                                'body_suffix': '<p class="programme__child-availability">' ~ tr(programme.getProgramme().getType() ~ '_is_related_to', {'%1': '<a class="block-link__link" href="' ~ path('topics_topic', {topic: programme.getRelatedByClasses()[0].getId()}) ~'" data-linktrack="episode_suggested_relationship">' ~ programme.getRelatedByClasses()[0].getTitle() ~ '</a></p>'}),
+                                                'body_suffix': '<p class="programme__child-availability">' ~ tr(programme.getProgramme().getType() ~ '_is_related_to', {'%1': '<a aria-label="' ~ tr(programme.getProgramme().getType() ~ '_is_related_to', {'%1': programme.getRelatedByClasses()[0].getTitle()}) ~ '" class="block-link__link" href="' ~ path('topics_topic', {topic: programme.getRelatedByClasses()[0].getId()}) ~'" data-linktrack="episode_suggested_relationship">' ~ programme.getRelatedByClasses()[0].getTitle() ~ '</a></p>'}),
```
Comment (BjornTwachtmann) [inline templates/find_by_pid/episode.html.twig]: Does the aria label here prefix or replace the text of the link?

Comment (cisco87) [reply templates/find_by_pid/episode.html.twig]: No it's the same text

File: src/Twig/HtmlUtilitiesExtension.php [hunk 2]
```diff
@@ -25,9 +30,15 @@ public function getFunctions()
             new Twig_Function('build_css_classes', [$this, 'buildCssClasses']),
             new Twig_Function('build_html_attributes', [$this, 'buildHtmlAttributes']),
             new Twig_Function('truncate', [$this, 'truncate']),
+            new Twig_Function('render_twig_file', [$this, 'renderTwigFile']),
         ];
     }
 
+    public function renderTwigFile($fileName, $arguments)
```
Comment (BjornTwachtmann) [inline src/Twig/HtmlUtilitiesExtension.php]: Might be worth dropping a comment here on the use case for this and why {{ include ... }} doesn't cut it.

**Pull request merged**